client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -41,28 +81,59 @@ class SpeedUpSiteSettings extends Component {
 			isSavingSettings,
 			translate,
 			jetpackVersionSupportsLazyImages,
+			siteAcceleratorSupported,
+			photonModuleActive,
+			assetCdnModuleActive,
+			togglingSiteAccelerator,
 		} = this.props;
 		const isRequestingOrSaving = isRequestingSettings || isSavingSettings;
+		const cdnStatus = photonModuleActive || assetCdnModuleActive;
 
 		return (
 			<div className="site-settings__module-settings site-settings__speed-up-site-settings">
 				<Card>
 					<FormFieldset className="site-settings__formfieldset">
 						<SupportInfo
-							text={ translate( 'Hosts your image files on the global WordPress.com servers.' ) }
-							link="https://jetpack.com/support/photon/"
-						/>
-						<JetpackModuleToggle
-							siteId={ selectedSiteId }
-							moduleSlug="photon"
-							label={ translate( 'Serve images from our servers' ) }
-							description={ translate(
-								'Jetpack will optimize your images and serve them from the server ' +
-									'location nearest to your visitors. Using our global content delivery ' +
-									'network will boost the loading speed of your site.'
+							text={ translate(
+								"Jetpack's global Content Delivery Network (CDN) optimizes " +
+									'files and images so your visitors enjoy ' +
+									'the fastest experience regardless of device or location.'
 							) }
-							disabled={ isRequestingOrSaving || photonModuleUnavailable }
+							link="http://jetpack.com/support/site-accelerator/"
 						/>
+						<FormSettingExplanation className="site-settings__feature-description">
+							{ translate(
+								'Load pages faster by allowing Jetpack to optimize your images and serve your images ' +
+									'and static files (like CSS and JavaScript) from our global network of servers.'
+							) }
+						</FormSettingExplanation>
+						<CompactFormToggle
+							checked={ !! cdnStatus }
+							disabled={
+								isRequestingOrSaving ||
+								photonModuleUnavailable ||
+								! siteAcceleratorSupported ||
+								togglingSiteAccelerator
+							}
+							onChange={ this.handleCdnChange }
+							toggling={ togglingSiteAccelerator }
+						>
+							{ translate( 'Enable site accelerator' ) }
+						</CompactFormToggle>
+						<div className="site-settings__child-settings">
+							<JetpackModuleToggle
+								siteId={ selectedSiteId }
+								moduleSlug="photon"
+								label={ translate( 'Speed up image load times' ) }
+								disabled={ isRequestingOrSaving || photonModuleUnavailable }
```

Comment:
I think it makes sense to disable all the controls inside this component when disabling any other one.

Reply:
👍 	Done that.